src/guide/jbs-jdk-bug-system.md

```diff
@@ -42,6 +42,12 @@ A few things to keep in mind when filing a new bug:
 
 To find out which component to use for different bugs, consult the [directory to area mapping](#directory-to-area-mapping).
 
+### Sensitive information (hs_err.log)
```

Comment:
Maybe "(hs_err.log)" -> "(e.g. hs_err.log)"?  While it's true that hs_err.log is the inadvertent source of sensitive information, we've had many cases of people just including that information in the description or comments directly (e.g. username in paths when describing test case).

Remediation is difficult.

Reply:
Right. Will fix.